geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -256,9 +258,12 @@ object SimpleFeatureTypes {
       if (ad.isBinTrackId) {
         options.put(OPT_BIN_TRACK_ID, "true")
       }
-      if (ad.isKeepStats) {
+      if (ad.isKeepStats()) {
```

Comment:
yeah, i prefer to have the `()` to indicate it's not a straight retrieval but it's actually doing a little processing. not sure if that's really a standard :P